lib/Parser/RegexRuntime.cpp

```diff
@@ -934,29 +934,36 @@ namespace UnifiedRegex
 
     void UnifiedRegexSwitchMixinForceAllInstantiations()
     {
-        {
-            SwitchMixin<10> x;
-            x.AddCase(0, 0);
-#if ENABLE_REGEX_CONFIG_OPTIONS
-            x.Print(0, 0);
-#endif
-        }
-        {
-            SwitchMixin<20> x;
-            x.AddCase(0, 0);
 #if ENABLE_REGEX_CONFIG_OPTIONS
-            x.Print(0, 0);
+#define SWITCH_FORCE_INSTANTIATION_PRINT x.Print(0, 0)
+#else
+#define SWITCH_FORCE_INSTANTIATION_PRINT
 #endif
+
+#define SWITCH_FORCE_INSTANTIATION(n)                       \
+        {                                                   \
+            SwitchMixin<n> x;                               \
+            x.AddCase(0, 0);                                \
+            SWITCH_FORCE_INSTANTIATION_PRINT;               \
         }
+
+        SWITCH_FORCE_INSTANTIATION(2);
+        SWITCH_FORCE_INSTANTIATION(4);
+        SWITCH_FORCE_INSTANTIATION(8);
+        SWITCH_FORCE_INSTANTIATION(16);
+        SWITCH_FORCE_INSTANTIATION(24);
```

Comment:
Why this instead of explicit instantiation? "template SwitchMixin<2>;" outside of a function should force instantiation of the class and its members.

Reply:
Yeah not sure. This is what was done previously, I just templatized it.